chains/manager.go

```diff
@@ -1212,6 +1214,7 @@ func (m *manager) createSnowmanChain(
 		Validators:    vdrs,
 		Params:        consensusParams,
 		Consensus:     consensus,
+		LiteSync:      m.LiteSyncPrimaryNetwork && commonCfg.Ctx.ChainID == constants.PlatformChainID,
```

Comment:
should we also check the network ID here?

Reply:
What would the restriction be?

Reply:
I don't see any issue with this.